document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/GetAccountsDocumentInfo.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.companieshouse.document.generator.accounts;
+
+import uk.gov.companieshouse.document.generator.interfaces.DocumentInfoService;
+import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfo;
+
+public class GetAccountsDocumentInfo implements DocumentInfoService {
```

Comment:
GetDocumentInfo was renamed in the interface to DocumentInfoService. Can you and @Mathsh discuss a consistent naming for the interface and implementations please?